.github/ISSUE_TEMPLATE/vsp-release-plan.md

```diff
@@ -1,59 +1,55 @@
-# Release Plan Template [Draft]
-This Release Plan Template is intended to be used to help ensure your new VSP product or feature is ready for launch and will deliver the desired user outcomes. In Phase 1, you will test your product to learn how it'll actually be used, what problems it might create, and then fix/adjust if necesssary prior to going live. In Phase 2, you will define the launch communications plan and post-launch KPIs. 
+name: VSP Initiative Release Plan Template (Draft)
 
-You do not need to complete a release plan for a product or feature update that does not:
+about: Release plan template for VSP products.
 
-- Add substantial new functionality to the Platform.
-- Impact user flows through tasks.
+title: [Initiative-Name]: Release Plan
 
-For your product or feature, make a copy of this template and check it into products/platform/<product-name>.
+---
+
+## Guidance
+
+_This Release Plan Template is intended to help ensure your new VSP product or feature is ready for launch and will deliver the desired user outcomes. In Phase 1, you will test your product to learn how it'll actually be used, what problems it might create, and then fix/adjust if necessary prior to going live. In Phase 2, you will define the launch communications plan and post-launch KPIs. You can close this ticket once you have completed Phase 2. Phase 3 is optional - you may use it to document post-launch results. 
```

Comment:
```suggestion
_This Release Plan Template is intended to help ensure your new VSP product or feature is ready for launch and will deliver the desired user outcomes. In Phase 1, you will test your product to learn how it'll actually be used, what problems it might create, and then fix/adjust if necessary prior to going live. In Phase 2, you will define the launch communications plan and post-launch KPIs. You can close this ticket once you have completed Phase 2. Phase 3 is optional - you may use it to document post-launch results._
```